tests/test_annotations.py

```diff
@@ -587,7 +587,7 @@ def test_self_reference(self, slots):
         @attr.s(slots=slots, auto_attribs=True)
         class A:
             a: "A"
-            b: typing.Optional["A"]  # noqa: will resolve below
```

Comment:
this breaks the CI.

generally speaking, thanks to `yesqa`, you don't have to manually remove `noqa`s

Reply:
(I have just noticed that this appeared to be a rogue yesqa action – still revert pls I've removed yesqa :))